docs/synthetics/faq.md

```diff
@@ -62,7 +65,7 @@ Short answer: because they are measuring somewhat different things.
 
 The HTTP monitor measures how long it takes to execute a single HTTP request and get a response. This includes performing a DNS lookup, establishing a TCP connection, sending of the request, and content transfer.  On the other hand, the Browser monitor measures all of that, plus how long it takes to load and render the whole page, which includes fetching of external resources references in that page (e.g., CSS resources, JavaScript resources, etc.).  If a user journey script is used with a Browser monitor then that is included as well.  Thus, it is expected that Browser monitors display higher latency.
 
-### You can whitelist Synthetics elastic IP according to your region
+### What are the IP addresses from which monitors are run?
 N. Virginia (us-east-1)    - 52.202.60.97   <br/>
 N. California (us-west-1)  - 184.72.63.129  <br/>
 Mumbai (ap-south-1)        - 15.207.239.77  <br/>
```

Comment:
Not part of the diff, but might be good to convert this into a table (Location/ServerName/IP) sometime in the future.